assets/libraries/serverlessfw.rego

```diff
@@ -35,3 +43,170 @@ get_service_name(document) = name{
+	"AWS::AutoScaling::AutoScalingGroup": "AutoScalingGroupName",
+	"AWS::RDS::DBInstance": "DBName",
+	"AWS::Batch::JobDefinition": "JobDefinitionName",
+	"AWS::CloudFront::Distribution": "",
+	"AWS::EC2::Instance": "",
+	"AWS::CloudTrail::Trail": "TrailName",
+	"AWS::Route53::HostedZone": "Name",
+	"AWS::KMS::Key": "",
+	"AWS::DocDB::DBCluster": "",
+	"AWS::Neptune::DBCluster": "",
+	"AWS::RDS::DBCluster": "DatabaseName",
+	"AWS::RDS::GlobalCluster": "",
+	"AWS::Redshift::Cluster": "DBName",
+	"AWS::CodeBuild::Project": "Name",
+	"AWS::Cognito::UserPool": "UserPoolName",
+	"AWS::Config::ConfigurationAggregator": "ConfigurationAggregatorName",
+	"AWS::IAM::Role": "RoleName",
+	"AWS::EC2::SecurityGroup": "GroupName",
+	"AWS::RDS::DBSecurityGroup": "",
+	"AWS::DirectoryService::MicrosoftAD": "Name",
+	"AWS::DirectoryService::SimpleAD": "Name",
+	"AWS::DMS::Endpoint": "",
+	"AWS::DynamoDB::Table": "TableName",
+	"AWS::EC2::Volume": "",
+	"AWS::EC2::NetworkAclEntry": "",
+	"AWS::EC2::Subnet": "",
+	"AWS::ECR::Repository": "RepositoryName",
+	"AWS::ECS::Service": "ServiceName",
+	"AWS::ECS::TaskDefinition": "",
+	"AWS::EFS::FileSystem": "",
+	"AWS::EKS::Nodegroup": "NodegroupName",
+	"AWS::Elasticsearch::Domain": "DomainName",
+	"AWS::ElastiCache::CacheCluster": "ClusterName",
+	"AWS::ElastiCache::ReplicationGroup": "",
+	"AWS::EMR::Cluster": "Name",
+	"AWS::EMR::SecurityConfiguration": "Name",
+	"AWS::EC2::SecurityGroupIngress": "GroupName",
+	"AWS::ECS::Cluster": "ClusterName",
+	"AWS::GameLift::Fleet": "Name",
+	"AWS::CodeStar::GitHubRepository": "RepositoryName",
+	"AWS::GuardDuty::Detector": "",
+	"AWS::Lambda::Function": "FunctionName",
+	"AWS::IAM::Group": "GroupName",
+	"AWS::IAM::ManagedPolicy": "ManagedPolicyName",
+	"AWS::IAM::User": "UserName",
+	"AWS::IAM::Policy": "PolicyName",
+	"AWS::IAM::AccessKey": "",
+	"AWS::IoT::Policy": "PolicyName",
+	"AWS::Kinesis::Stream": "Name",
+	"AWS::Lambda::Permission": "",
+	"AWS::MSK::Cluster": "ClusterName",
+	"AWS::EC2::Route": "",
+	"AWS::S3::Bucket": "BucketName",
+	"AWS::S3::BucketPolicy": "",
+	"AWS::SageMaker::NotebookInstance": "NotebookInstanceName",
+	"AWS::SageMaker::EndpointConfig": "EndpointConfigName",
+	"AWS::SDB::Domain": "",
+	"AWS::SecretsManager::Secret": "Name",
+	"AWS::EC2::SecurityGroupEgress": "",
+	"AWS::GlobalAccelerator::Accelerator": "Name",
+	"AWS::EC2::EIP": "",
+	"AWS::SNS::TopicPolicy": "",
+	"AWS::SNS::Topic": "TopicName",
+	"AWS::SQS::QueuePolicy": "",
+	"AWS::SQS::Queue": "QueueName",
+	"AWS::CloudFormation::Stack": "",
+	"AWS::CloudFormation::StackSet": "StackSetName",
+	"AWS::AutoScaling::LaunchConfiguration": "LaunchConfigurationName",
+	"AWS::EC2::VPC": "",
+	"AWS::EC2::VPCGatewayAttachment": "",
+	"AWS::EC2::FlowLog": "",
+	"AWS::NetworkFirewall::Firewall": "FirewallName",
+	"AWS::WAF::WebACL": "Name",
+	"AWS::CertificateManager::Certificate": "",
+	"AWS::Serverless::HttpApi": "",
+	"AWS::Serverless::Api": "",
+	"AWS::Serverless::Function": "FunctionName",
+}
```

Comment:
```suggestion
```
Imho, there is no need to define `resourceFieldName` again. We can use it from CF library